strawman-Array.prototype.last.md

```diff
@@ -0,0 +1,62 @@
+# Array.prototype.last
+A proposal for an ECMAScript native method
+
+Code
+---
+
+    //everything in 6 lines
+    Array.prototype.last = function () {
```

Comment:
@shovon:  So it would be equivalent to `.find`? Then it should probably me renamed to `.findLast`, or similar.